docs/design/proposals/adoption/resource-adoption.md

```diff
@@ -0,0 +1,201 @@
+# Resource Adoption 
+
+# The Problem
+
+New ACK users typically have existing AWS resources in their accounts. These resources were not originally created by an ACK service controller. While ACK can create new resources, getting ACK service controllers to take over management of an existing resource is not yet supported.
+
+The current method for migrating management of resources over to ACK requires users to store the current state of their applications, delete the resources, and re-create them using ACK. This process introduces places for bugs, loss of the state of existing deployments and downtime in production workloads.
+
+ACK aims to provide the ability to “adopt” existing resources into your Kubernetes environment. Given the user can provide an IAM role which has access to existing resource, and that ACK operators exist for each resource’s associated service, the ACK controller will be able to determine the current specification and status of the AWS resource and reconcile said resource as if it were created natively in k8s.
```

Comment:
s/ACK operators/ACK controllers/gI :)

Reply:
s/as if it were created natively in k8s/as if the ACK controller had originally created it/